report/3.Theory/1.TrackingObjects.tex

```diff
@@ -0,0 +1,25 @@
+\section{Predicting a moving objects location}
+In order to predict a moving objects location a keen observation to be made is whether the object is flying i.e. if the object is following a ballistic trajectory or whether the object is driving i.e. following a linear trajectory. 
+In the following section we will elaborate on these distinctions in order to gain a deeper understanding on the subject and to be able to complete our objectivs. 
+\subsection{Linear trajectory}
+Given that the target follows a linear trajectory a simple method to calculate the new location of the target can be applied. 
+By using the image processing method as descibed in Section~\ref{} locating the target is considered irrelevent for this subsection. 
+In order to predict our targets location we first need to understand some basic physics of kinematics. 
+That is given an object is traveling with constant speed, the distance traveled by the object can be described as follows:
+\begin{equation}
+s = v \cdot t
+\end{equation}
+where $ s $ is the distance traveled, $ v $ is the velocity and $ t $ is the time passed.
+Hence an object traveling with $ 1 m/s $ for $ 1 s $ will have traveled 1 meter. 
+We can thereby calculate the speed of the target and use image reginition to establish the direction calculate the future position of the target. 
+
+%%%%%%%%%%%%%%%%%%%%%%
+% Der mangler stadig %
+% at blive skrevet om%
+% hvordan det kan    %
+% bruges i vores     %
+% system.            %
+%%%%%%%%%%%%%%%%%%%%%%
+
+\subsection{Ballistic trajectory}
+Since we do not just want to track an object flying in front of our device, we do also want to predict where it will be after a period of time. 
```

Comment:
The "since" at the start of the sentence seems to be misplaced. I believe it should be removed entirely.

Reply:
evt noget ala (ikke verbatim.. jeg smider bare keypoints ud) "since we want to lead the object, rather than follow it, prediction is needed. Whether this is done by manually implementing an algorithms or by using machine learning, the mathmatical principles behind trajectories and kinematics are relevant, as these principles are required to figure out which variables are relevant for either an algoritm or from a machine intelligence perspective" og så ændr subsection title til "takeaway" x) så følger den de andre